deflate_medium.c

```diff
@@ -273,15 +273,15 @@ ZLIB_INTERNAL block_state deflate_medium(deflate_state *s, int flush) {
         /* move the "cursor" forward */
         s->strstart += current_match.match_length;
 
-        if (bflush)
+        if (UNLIKELY(bflush))
```

Comment:
Is it ever possible that when `bflush=1` this will never get hit?

Reply:
The question is rather how often do we call this with bflush=1? And do we want to optimize for that?

Reply:
I see ok